### DISCUSSION TITLE
can custom voice cloning be supported ?

### DISCUSSION THREAD

**Discussion by dongavel** [suno-ai/bark#332] [Q&A]
I have some high-quality audio of different characters recorded by professional voice actors, and how can I change it to the npz file format that bark can use? is there some tool or functions in bark can help me to do this?

**Comment by adriens** [comment]
> high-quality audio of different characters recorded by professional voice actors, 

:star_struck:

**Comment by Merkoso** [comment]
Totally same question for me

**Comment by drishyakarki** [comment]
You can use bark with coqui-tts for the custom voice cloning.....it gives decent results

**Reply by ROBERT-MCDOWELL** [reply]
coqui-bark does not have functions to convert wav to npz.

**Comment by jakubMitura14** [comment]
Similar question from me and if npz would be available how to use them in bark
